tests/src/python/test_provider_ogr_gpkg.py

```diff
@@ -1230,69 +1230,6 @@ def testTransaction(self):
         self.assertEqual(len([f for f in vl2_external.getFeatures(QgsFeatureRequest())]), 1)
         del vl2_external
 
-    def testJson(self):
```

Comment:
Was that deleted on purpose?

Reply:
@m-kuhn, yes see my commit c908306. It appears this test was inappropriately backported in 2831a02 (a commit about identifier quoting, so json stuff isn't related). It didn't pass when running test_provider_ogr_gpkg.py in 3.4 branch with GDAL 2.4 (the failure is not seen on Travis since it must use an older versio)

Reply:
ah, ok, unrelated to this commit but on purpose